_data/sites.json

```
@@ -17600,6 +17600,16 @@
             "support.signal.org"
         ]
     },
+    {
+        "name": "SignalRGB",
+        "url": "https://signalrgb.com/account/delete-account/",
+        "difficulty": "easy",
+        "notes": "Go to [Delete page](https://signalrgb.com/account/delete-account/) and request to delete account.",
```

```suggestion
        "difficulty": "easy",
```
Since you provided a direct link, I think we can leave the default notes for `easy`